scylla/src/statement/prepared_statement.rs

```diff
@@ -74,19 +74,19 @@ impl PreparedStatement {
     /// Computes the partition key of the target table from given values
     /// Partition keys have a specific serialization rules.
     /// Ref: https://github.com/scylladb/scylla/blob/40adf38915b6d8f5314c621a94d694d172360833/compound_compat.hh#L33-L47
-    pub fn compute_partition_key(
+    pub fn compute_partition_key<'a>(
```

Comment:
I've added this explicit lifetime because the closure below required an explicit lifetime.